keyboards/bioi/ble.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 Basic I/O Instruments(Scott Wei) <scot.wei@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <stdbool.h>
+#include "host_driver.h"
+#include "host.h"
+
+
+typedef union {
+  uint32_t raw;
+  struct {
+    bool init : 1;
+  };
+} user_config_t;
+
+user_config_t ble_config;
```

Comment:
This shouldn't be here, this should be an extern, and the actual structure defined in a c file, or you may run into compiler issues
```suggestion
extern keyboard_config_t ble_config;
```